app.js

```diff
@@ -0,0 +1,88 @@
+const fs = require('fs')
+const fetch = require('node-fetch')
+const colors = require('colors')
+
+console.log("Jessica Castillo / Proyecto MDLINKS /Laboratoria CDMX009")
+
+function getContentString() {
+
+    let index = process.argv.indexOf("--file")
+    if (index < 0) return console.log("You need to use a valid uri flag --file")
+    let uri = process.argv[index + 1]
+    let string = fs.readFileSync(uri, 'utf8')
+    return string
+}
+
+function getLinks(string) {
+
+    const getArray = ("Text: ", string.toString());
+    let regExpression = /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g;
+    return getArray.match(regExpression);
+}
+
+function validateLinks(arrayLinks) {
+        let i=0
+        let work = 0
+        let broke = 0
+
+        for( i=0 ;i <arrayLinks.length;i++){
+
+            fetch(arrayLinks[i]).then(function(response) {
+                if (response.status === 200) {
+                    work ++;
+        
+                }else if (response.status == 404) {
+                    broke ++;
+    
+        
+                }else {
+                    console.log('error', response.status);
+                  }
+                console.log(`✔ Total Links: ${arrayLinks.length}`.brightYellow);
+                console.log(`✔ Total work Links: ${work}`.green);
+                console.log(`✖ Total Broken links: ${broke}`.red);
+
+            }
+          )
+        
+        };
```

Comment:
Ten presente la identación, recuerda que también puedes utilizar foreach ó map
```suggestion
        for( i=0 ;i <arrayLinks.length;i++){
               fetch(arrayLinks[i]).then(function(response) {
                if (response.status === 200) {
                    work ++;
                    }else if (response.status == 404) {
                    broke ++;
                }else {
                    console.log('error', response.status);
                  }
                console.log(`✔ Total Links: ${arrayLinks.length}`.brightYellow);
                console.log(`✔ Total work Links: ${work}`.green);
                console.log(`✖ Total Broken links: ${broke}`.red);

            }
          )
        
        };
```